vpr/src/place/place.cpp

```diff
@@ -1047,9 +1202,11 @@ static void placement_inner_loop(float t,
                                  ClusteredPinTimingInvalidator* pin_timing_invalidator,
                                  const PlaceDelayModel* delay_model,
                                  PlacerCriticalities* criticalities,
+                                 PlacerSetupSlacks* setup_slacks,
                                  MoveGenerator& move_generator,
                                  t_pl_blocks_to_be_moved& blocks_affected,
-                                 SetupTimingInfo* timing_info) {
+                                 SetupTimingInfo* timing_info,
+                                 const t_place_algorithm& place_algorithm) {
```

Comment:
This overrides `placer_opts`? It's not clear how this works.

Reply:
```placement_inner_loop()``` is called for both normal annealing and the quench. The normal annealing now uses ```placer_opts.place_algorithm``` and the quench now uses ```placer_opts.place_quench_algorithm```. Instead of creating a new separate function, I have a new parameter ```place_algorithm``` passed in to override ```placer_opts.place_algorithm```